contracts/Tokens/VTokens/VToken.sol

```diff
@@ -302,6 +328,32 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return getCashPrior();
     }
 
+    /**
+     * @notice A admin function to set new threshold of block difference after which funds will be sent to the protocol share reserve
+     * @param newReduceReservesBlockDelta_ block difference value
+     */
+    function setReduceReservesBlockDelta(uint256 newReduceReservesBlockDelta_) external returns (uint) {
+        // Check caller is admin
+        if (msg.sender != admin) {
+            return fail(Error.UNAUTHORIZED, FailureInfo.SET_REDUCE_RESERVES_BLOCK_DELTA_OWNER_CHECK);
+        }
```

Comment:
I think we should use ACM here because the function is setting a param, not setting a contract.

```
ensureAllowed("setReduceReservesBlockDelta(uint256)");
```